docs/pipeline-standards.md

```diff
@@ -15,241 +15,244 @@ Libuit KG, Guthrie J, Ambrosio F, Kapsak C, Unal Gultekin, Holmes J, Wright S, N
-- To verify this standard, the reviewer should look for explicit statements or documentation expressing a commitment to ongoing maintenance, with evidence of regular updates, improvements, user support, and easily accessibly means of contact. Bi-annual re-verification will require the reviewer to reach out to the point of contact, and update if necessary.
+There should be a clear summary in the Git README pointing to which containerisation method has been chosen and instructions for how a lab can install this pipeline and/or where docker images are available, e.g. dockerhub and quay for containers; and where conda packages are available, e.g. anaconda and bioconda (cross-referenced with Installation Instructions).
 
-### Conflict of Interest Statement
-_Have all potential conflicts of interest been disclosed?_
+When bioinformatics software is containerised, there should be a supplementary document justifying, where appropriate, why a particular version was chosen for analysis. Reasons for choosing specific pipeline versions include but are not limited to ensuring functionality by selecting compatible dependencies and major differences in functionality and features between pipeline versions.
+Similarly, if a pipeline version within the container changes, this should be appropriately version-controlled, outlining any expected effects on results, documented in a CHANGELOG. This could also provide a useful way for prompting improvements during the code review process, e.g. questioning why pipeline with an upcoming end-of-life was chosen during development.
 
-Stating potential conflicts of interest regarding software authors in public health bioinformatics promotes transparency and integrity in scientific research. This practice enables users to make informed decisions about the software they utilize, ensuring unbiased results and fostering trust within the research community. Conflict of interest is defined as any factor which renders an author, co-author, or collaborative team unable to (potentially or otherwise) perform an independent peer review or evaluation pertaining to a study. Examples of conflict of interest include but are not limited to commercial, personal, political, or religious interests. When developing bioinformatics (for public health) software, pipelines, or methods, any conflict of interest should be disclosed by the responsible authors to ensure independent peer review and testing of code has been carried out prior to publication. Some conflict of interest statements may be waived if an author can demonstrate they are able to perform an impartial code review. For example, JOSS suggests that if two co-authors did not ever truly collaborate, this might mean a co-author is a suitable selection for code review. 
+**To adhear this best practice:** Implement pipeline components within Docker containers or distribute them as Conda packages.
```

Comment:
I will add here something related to the `There should be a clear summary in the Git README`